testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -1058,6 +1064,12 @@ def test_UnWrapFlag_residues(self, ag, ref_Unwrap_residues):
         assert_almost_equal(ag.moment_of_inertia(unwrap=True, compound='residues'), ref_Unwrap_residues['MOI'], self.prec)
         assert_almost_equal(ag.asphericity(unwrap=True, compound='residues'), ref_Unwrap_residues['Asph'], self.prec)
 
+    def test_UnWrapFlag_residues_unordered(self, unordered_ag, ref_Unwrap_residues):
+        assert_almost_equal(unordered_ag.center_of_geometry(unwrap=True, compound='residues'), ref_Unwrap_residues['COG'], self.prec)
```

Comment:
These lines could be split.

Reply:
I rReally just kept them long to conform to how all other such tests are. I can split (shall I also split the analogous ones in neighboring tests?)